### DISCUSSION TITLE
Error Message When Uploading a Video

### DISCUSSION THREAD

**Discussion by MrCamotes** [ant-media/Ant-Media-Server#6249] [Announcements]
I keep getting an error message when I try to upload a video to VOD

**Comment by burak-58** [comment]
Let me convert this to a github discussion. Please give more information so that we can understand the issue better.

**Comment by muratugureminoglu** [comment]
Hi @MrCamotes 

If you share the error message with us, we can help you quickly.

Please provide the following.

- File size
- Log file
- Error message
